index.js

```diff
@@ -0,0 +1,84 @@
+const express = require("express");
+const knex = require("knex");
+const knexConfig = require("./knexfile");
+const server = express();
+
+server.use(express.json());
+
+const db = knex(knexConfig.development);
+
+server.post("/api/cohorts", (req, res) => {
+  db("cohorts")
+    .insert(req.body)
+    .then(ids => {
+      db("cohorts")
+        .where({ id: ids[0] })
+        .then(cohort => {
+          res.status(201).json(cohort);
+        });
+    })
+    .catch(err => res.status(500).json(err));
+});
+
+server.get("/api/cohorts", (req, res) => {
+  db("students")
+    .then(results => {
+      res.status(200).json(results);
+    })
+    .catch(err => {
+      res.status(500).json(err);
+    });
+});
```

Comment:
I'm not getting any data being returned here. In your function here, you are attempting to call 'students' instead of 'cohorts', and it is returning 0 results back in an array